ml-agents/mlagents/trainers/torch/layers.py

```diff
@@ -82,3 +84,62 @@ def lstm_layer(
                         forget_bias
                     )
     return lstm
+
+
+class MemoryModule(torch.nn.Module):
```

Comment:
Not sure if the MemoryModule interface is needed if LSTM is the only implementation of it. Do you think there will be more MemoryModules in the future?

Reply:
[Yes](https://github.com/Unity-Technologies/ml-agents/pull/4374) 😛 OFC still experimental 

Also things like Transformers, GRU, etc. Not sure how many of these things will make it to `master`, but I assume we're going to want this for experimentation.